### ISSUE TITLE
Android Autofill framework

### ISSUE THREAD

**Issue by neekless** [bromite/bromite#348]
**Is your feature request related to privacy?**
No but I believe it will fix the Autofill issues people raised in https://github.com/bromite/bromite/issues/230

**Is there a patch available for this feature somewhere?**
It's not exactly a patch but the Autofill Framework requires [Webview components](https://github.com/chromium/chromium/blob/master/android_webview/java/src/org/chromium/android_webview/AwAutofillProvider.java), thus monochrome.

**Describe the solution you'd like**
I referenced Brave browser which ships monochrome builds and works with the Autofill Framework.

I then tested it with your Bromite System Webview and [Lynket](https://github.com/arunkumar9t2/lynket-browser), which allows me to open links in custom tabs using a browser of my choice, including System Webview (thus Bromite system Webview).
![Screenshot_Settings_20190805-161329](https://user-images.githubusercontent.com/22426822/62449235-4939ae00-b759-11e9-9a41-bd292e52c485.png)
![Screenshot_Lynket_20190805-161317](https://user-images.githubusercontent.com/22426822/62449238-4b037180-b759-11e9-9f97-514d300a8e06.png)

I then opened a link containing a login page using Bromite System Webview through Lynket and indeed the Autofill Framework pops up.
![Screenshot_Lynket_20190805-155644](https://user-images.githubusercontent.com/22426822/62448956-8cdfe800-b758-11e9-9b96-abf8376f76da.png)

Regarding Chromium showing the same behavior mentioned in https://github.com/bromite/bromite/issues/230, the Autofill Framework doesn't work with your Chromium builds because 1. it's not monochrome and 2. the package name is not `org.chromium.chrome`, which Bitwarden detects for the Autofill Framework.

In conclusion, Bromite can support the Android Autofill Framework if the Webview components are included. Is there any reason you stopped building monochrome?

**Comment by csagan5** [comment]
> 2. the package name is not `org.chromium.chrome`, which Bitwarden detects for the Autofill Framework.

That is not true, Bitwarden added Bromite support a while ago: https://github.com/bitwarden/mobile/search?q=org.bromite.bromite&unscoped_q=org.bromite.bromite

> the Autofill Framework doesn't work with your Chromium builds because 1. it's not monochrome

While you can conclude from your tests that it works with the Webview, you cannot conclude that it works only with a webview or Monochrome.

As explained in the issue you linked there is in Bromite a patch that disables the cloud account integration of autofill and in another patch the autofill settings are set to disabled by default.

Have you tried enabling Bitwarden as an accessibility app? In next release I will make this easier by letting the autofill profile be enabled by default.

I noticed that it displays a Bitwarden prompt for `org.bromite.bromite` and after cancelling that it allows using the autofill entries.

**Comment by csagan5** [comment]
Fixed in `76.0.3809.100`.
